src/08-tasks/tasks.apib

```diff
@@ -42,6 +42,12 @@ Get a list of tasks.
                 + work_type (object, nullable)
                     + type: `workType` (string)
                     + id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string)
+                + assigned_to (object)
```

Comment:
Comparing to other API endpoints it would seem that `assignee` would be a better fit. See `invoicee`, `creator`, `customer`, ...

Does that make sense here too?